<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string comment="Window Title">Wegweiser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_20">
    <item>
     <layout class="QGridLayout" name="userInputLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="routesLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="routeBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Routes</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QListView" name="routeBoxRouteList"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="routeBoxButtonLayout">
             <item>
              <widget class="QPushButton" name="routeBoxButtonPushAdd">
               <property name="text">
                <string>Push-Add</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="routeBoxButtonAddNode">
               <property name="text">
                <string>Add @ selected</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="routeBoxButtonDeleteNode">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="routeBoxButtonUp">
               <property name="text">
                <string>Up</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="routeBoxButtonDown">
               <property name="text">
                <string>Down</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="nodeBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Route Nodes</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QListView" name="nodeBoxNodeList"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="nodeBoxButtonLayout">
             <item>
              <widget class="QPushButton" name="nodeBoxButtonPushAdd">
               <property name="text">
                <string>Push-Add</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="nodeBoxButtonAddNode">
               <property name="text">
                <string>Add @ selected</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="nodeBoxButtonDeleteNode">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="nodeBoxButtonUp">
               <property name="text">
                <string>Up</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="nodeBoxButtonDown">
               <property name="text">
                <string>Down</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="settingsBox">
        <property name="title">
         <string>Settings</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QGroupBox" name="currentRouteBox">
           <property name="title">
            <string>Route</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="routeNameLayout">
              <item>
               <widget class="QLabel" name="routeNameLabel">
                <property name="text">
                 <string>Name</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::AutoText</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="routeNameLineEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="routeLabelLayout">
              <item>
               <widget class="QLabel" name="routeLabelLabel">
                <property name="text">
                 <string>Label</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="routeLineEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="routeColorLayout">
              <item>
               <widget class="QLabel" name="routeColorLabel">
                <property name="text">
                 <string>Color</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="routeColorButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>125</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="routeStyleLayout">
              <item>
               <widget class="QLabel" name="routeStyleLabel">
                <property name="text">
                 <string>Style</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="routeStyleComboBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>125</width>
                  <height>0</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>Style1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Style2</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="routeNodeOrderLayout">
              <item>
               <widget class="QLabel" name="routeNodeOrderLabel">
                <property name="text">
                 <string>Show Node Order</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="routeNodeOrderCheckBox">
                <property name="maximumSize">
                 <size>
                  <width>15</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="routeDirectionLayout">
              <item>
               <widget class="QLabel" name="routeDirectionLabel">
                <property name="text">
                 <string>Show Direction</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="routeDirectionCheckBox">
                <property name="maximumSize">
                 <size>
                  <width>15</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="currentNodeBox">
           <property name="title">
            <string>Current Node</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="nodeNameLayout">
              <item>
               <widget class="QLabel" name="nodeNameLabel">
                <property name="text">
                 <string>Name</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::AutoText</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="nodeNameLineEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="nodeLabelLayout">
              <item>
               <widget class="QLabel" name="nodeLabelLabel">
                <property name="text">
                 <string>Label</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="nodeLabelLineEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="nodeColorLayout">
              <item>
               <widget class="QLabel" name="nodeColorLabel">
                <property name="text">
                 <string>Color</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="nodeColorButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>125</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="nodeStyleLayout">
              <item>
               <widget class="QLabel" name="nodeStyleLabel">
                <property name="text">
                 <string>Style</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="nodeStyleComboBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>125</width>
                  <height>0</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>Style1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Style2</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="nodeShowOrderLayout">
              <item>
               <widget class="QLabel" name="nodeShowOrderLabel">
                <property name="text">
                 <string>Show Node Order</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="nodeShowOrderCheckBox">
                <property name="maximumSize">
                 <size>
                  <width>15</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="nodeInvisibleLayout">
              <item>
               <widget class="QLabel" name="nodeInvisibleLabel">
                <property name="text">
                 <string>Invisible</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="nodeInvisibleCheckBox">
                <property name="maximumSize">
                 <size>
                  <width>15</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="pictureLayout">
      <item>
       <widget class="QScrollArea" name="pictureScrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="pictureScrollAreaContent">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>346</width>
           <height>532</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0" colspan="2">
           <widget class="MapView" name="picture">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>23</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapView</class>
   <extends>QGraphicsView</extends>
   <header>ui/mapview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
